dipy/reconst/recspeed.pyx

```diff
@@ -278,7 +316,7 @@ def local_maxima(cnp.ndarray odf, cnp.ndarray edges):
 @cython.wraparound(False)
 @cython.boundscheck(False)
 cdef void _cosort(double[::1] A, cnp.npy_intp[::1] B) nogil:
-    """Sorts A inplace and applies the same reording to B"""
+    """ Sorts `A` in-place and applies the same reordering to `B`"""
```

Comment:
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard seems to prefer no space, so I did that instead.